appveyor/install-miniconda.ps1

```diff
@@ -82,6 +86,9 @@ $env:PATH = "${env:PYTHON};${env:PYTHON}\Scripts;" + $env:PATH
 conda config --set always_yes true
 conda config --add channels defaults
 
+# Install the build and runtime dependencies of the project.
+conda update -q conda=$env:CONDA_VERSION

```

Comment:
We want to update what came in the environment? I guess. Or does ``install`` also update the dependencies? (I should really start using conda locally to get to know it better).